### DISCUSSION TITLE
A timed re-centering event?

### DISCUSSION THREAD

**Discussion by awjohnso** [swiftDialog/swiftDialog#435] [Ideas]
When giving the user the ability to move the window out of the way, so they can save their work for example, it's easy for them to then ignore the Swift Dialog window. It would be nice if after a time interval that the window could recenter it self?

Right now I think I have to quit and relaunch it if I'm not mistaken.

**Comment by bartreardon** [comment]
you can send a `position: center` command to the command file if you want it to be moved back to the middle of the screen. described in https://github.com/swiftDialog/swiftDialog/wiki/Updating-Dialog-with-new-content

**Comment by awjohnso** [comment]
Awesome! Thank you!! I guess I missed that when going through the list of commands. Just tested and it works as expected.
